depext.ml

```diff
@@ -121,12 +125,13 @@ let depexts ~with_tests ~with_docs opam_packages =
     fatal_error
       "This version of opam-depext requires opam 2.0.0~beta5 or higher";
   let c =
+    List.iter cannot_escape_single_quotes opam_packages;
```

Comment:
I have now checked - single quotes can be escaped within a single-quoted string, they have to be substituted with `'"'"'`  or `'\''` (in both cases, the idea is that you close the single-quoted string, producing a single quote character, and then resume a single-quoted string - the see [1003.1-2017](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_03)). The additional quote marks are then removed by quote elimination in the final step of word expansion.